lang/en.json

```diff
@@ -208,52 +208,52 @@
         "EtrackerTagHelp": "",
         "Except": "Except",
         "ExportDraft": "Export draft",
         "ExportX": "Export %s",
         "FacebookPixelTagDescription": "The Facebook Pixel is a web analytics and advertising service offered by Facebook.",
-        "FacebookPixelTagHelp": "This tag lets you track website pageviews in your Facebook Ads account. To obtain the Pixel ID please log in to your Facebook Ads account.",
+        "FacebookPixelTagHelp": "This tag lets you track website pageviews in your Facebook Ads account. Obtain the Pixel ID by logging in on your Facebook Ads account.",
         "FacebookPixelTagName": "Facebook Pixel",
         "Filter": "Filter",
         "FireDelay": "Fire delay",
-        "FireDelayHelp": "Lets you delay the execution of this tag. This can be useful for example if you want to allow some time for a blocking trigger to stop the tag from being executed. The value is expected in milliseconds where 1000ms are equal to one second.",
+        "FireDelayHelp": "Lets you delay the execution of this tag. Can be useful if you for example want to allow some time for a blocking trigger to stop the tag from being executed. The value is expected in milliseconds, and 1000 ms is one full second.",
         "FireLimit": "Fire limit",
-        "FireLimitHelp": "The fire limit defines how often this tag will be executed. When set to '%1$s', then the tag will be executed each time any of the assigned fire triggers is triggered. When set to '%2$s', then the tag will be executed max during a single page view. '%3$s' means the tag will not be executed again within the next 24 hours. Opening a new tab or window may cause a new session as well. '%4$s' means the tag will be only executed once. Please note for this to work some information will be stored in the browser based on the name of this tag. This means if you choose '%4$s' and you change the name of this tag, the tag may be executed again for everyone.",
-        "FireTriggerHelp": "This tag will be executed every time any of these triggers are triggered. Not all of them have to be triggered in order for the tag to be executed. Want to execute the trigger only once per page view or session? You can limit how often the tag should be executed in the advanced settings by changing the \"%1$s\".",
+        "FireLimitHelp": "The fire limit defines how often this tag is executed. When set to '%1$s', the tag is executed each time any of the assigned fire triggers are triggered. When set to '%2$s', then the tag will be executed max during a single pageview. '%3$s' means the tag will not be executed again within the next 24 hours. Opening a new tab or window may causes a new session too. '%4$s' means the tag will only be executed once. Please note for this to work some info will be stored in the web browser based on the name of this tag. This means if you choose '%4$s' and you change the name of this tag, the tag may be executed again for everyone.",
```

Comment:
```suggestion
        "FireLimitHelp": "The fire limit defines how often this tag is executed. When set to '%1$s', the tag is executed each time any of the assigned fire triggers are triggered. When set to '%2$s', then the tag will at most be executed once during a pageview. '%3$s' means the tag will not be executed again within the next 24 hours. Opening a new tab or window may causes a new session too. '%4$s' means the tag will only be executed once. Please note for this to work some info will be stored in the web browser based on the name of this tag. This means if you choose '%4$s' and you change the name of this tag, the tag may be executed again for everyone.",
```
?

Reply:
`%2$s` is being replaced with `Once per pageview`, so you suggestions sounds fine.